src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionVisibilityOptions.tag

```diff
@@ -12,8 +12,9 @@
 <c:set var="FEEDBACK_STUDENTS"><%=FeedbackParticipantType.STUDENTS.name()%></c:set>
 <c:set var="FEEDBACK_INSTRUCTORS"><%=FeedbackParticipantType.INSTRUCTORS.name()%></c:set>
 
-<div class="col-sm-12 margin-bottom-15px padding-15px background-color-light-green">
+<div class="col-sm-12 margin-bottom-15px padding-15px <%= fqForm.isQuestionHasResponses() ? "alert alert-danger" : "background-color-light-green" %>">
     <div class="margin-bottom-7px">
+        <c:if test="${fqForm.questionHasResponses}"><h4>Changing the visibility after collecting responses is not recommended.</h4></c:if>
```

Comment:
@damithc Changed the code to this:
```HTML
<c:if test="${fqForm.questionHasResponses}">
    <h4>Changing the visibility after collecting responses is not recommended.</h4>
    <p><em>Reason: The existing responses were submitted under the 'promise' of a certain visibility and changing the visibility later 'breaks' that promise.</em></p>
</c:if>
```
Looks like this now:

<img width="1115" alt="screen shot 2017-06-24 at 4 04 35 pm" src="https://user-images.githubusercontent.com/18088721/27507875-e3513b84-58f6-11e7-8322-99ed81ba80f9.png">

Is this okay? Or can it do without the `<em>` tag? Any other cosmetic suggestions for this line?